cluster/src/main/java/io/atomix/cluster/AtomixClusterBuilder.java

```diff
@@ -328,7 +330,10 @@ public AtomixClusterBuilder withMulticastAddress(Address address) {
    */
   @Deprecated
   public AtomixClusterBuilder setBroadcastInterval(Duration interval) {
-    config.getMembershipConfig().setBroadcastInterval(interval);
+    GroupMembershipProtocolConfig protocolConfig = config.getProtocolConfig();
+    if (protocolConfig instanceof PhiMembershipProtocolConfig) {
+      ((PhiMembershipProtocolConfig) protocolConfig).setHeartbeatInterval(interval);
```

Comment:
might be a bit cleaner for GroupMembershipProtocolConfig to implement a generic interface method setBroadcastInterval?

Reply:
This is just for backwards compatibility